crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -156,6 +217,132 @@ pub(crate) async fn farm(
     multi_farming.wait().await
 }
 
+const BENCH_FARMER_METADATA: FarmerMetadata = FarmerMetadata {
+    record_size: PIECE_SIZE as u32 - 96, // PIECE_SIZE - WITNESS_SIZE
+    recorded_history_segment_size: PIECE_SIZE as u32 * 256 / 2, // PIECE_SIZE * MERKLE_NUM_LEAVES / 2
+    max_plot_size: 100 * 1024 * 1024 * 1024 / PIECE_SIZE as u64, // 100G
+};
+
+pub(crate) async fn bench(
```

Comment:
Looks like I missed, but what is `fn bench` doing in `farm.rs`? It should have been `bench.rs`, the file was created specifically for `farm` command because it was non-trivial.